chg: dev: Add more information for failure messages

baraserg opened the pull request:
Additional information when image fails to initialize

carlos-jenkins commented on lib/topology_docker_openswitch/openswitch.py:
```diff
@@ -121,9 +121,10 @@ def cur_cfg_is_set():
-        return 0
+        raise Exception('Image returned an unexpected structure '
+            'and index is out of range, please verify your image')
 
```
It should be:

``return False``

Do not kill the process here! The index out of range must be treated as "not yet ready" instead of an "everything went wrong, kill the process".

baraserg replied:
Thanks Carlos, I fixed the issue

carlos-jenkins commented on lib/topology_docker_openswitch/openswitch.py:
```diff
@@ -121,9 +121,9 @@ def cur_cfg_is_set():
     response = loads(sock.recv(4096))
     try:
-        return response['result'][0]['rows'][0]['cur_hw'] == 1
+        return response['result'][0]['rows'][0]['cur_hw']
```
Why the boolean evaluation was removed? Please fix this.

baraserg replied:
I added the verification in lines 193 and 194, I removed it so I can pass to the log output the obtained value

carlos-jenkins commented:
@baraserg can you please fix the last issue so we can merge this?

baraserg commented:
@carlos-jenkins I added the verification in lines 193 and 194, I removed it so I can pass to the log output the obtained value, let me know if this works

baraserg commented:
Information was added on different pull request, closing this one
